File: src/main/java/net/ndrei/teslacorelib/gui/IGuiContainerPiecesProvider.java
package net.ndrei.teslacorelib.gui;

import java.util.List;

/**
 * Created by CF on 2016-12-21.
 */
public interface IGuiContainerPiecesProvider {
    List<IGuiContainerPiece> getGuiContainerPieces(BasicTeslaGuiContainer container);
}

File: src/main/kotlin/net/ndrei/teslacorelib/gui/IGuiContainerPiecesProvider.kt
package net.ndrei.teslacorelib.gui

/**
 * Created by CF on 2017-06-28.
 */
interface IGuiContainerPiecesProvider {
    fun getGuiContainerPieces(container: BasicTeslaGuiContainer<*>): MutableList<IGuiContainerPiece>
}